src/cms/templates/push_notifications/push_notification_form.html

```diff
@@ -49,13 +54,16 @@ <h1 class="heading">
                 <div id="tab-{{ form.language.value }}" class="language-tab-content w-full mb-4 rounded border-2 border-blue-500 bg-white flex-auto {% if form.language.value|add:0 != language.id|add:0 %}hidden{% endif %}">
                     <div class="w-full p-4">
                         {{push_notification_translation_form.errors}}
+                        
                         <div class="w-full p-4 mb-4 rounded border border-solid border-gray-200 shadow bg-white">
                             <label for="{{ form.title.id_for_label }}" class="block mb-2 font-bold cursor-pointer">{{ form.title.label }}</label>
                             {% render_field form.title class="appearance-none block w-full bg-gray-200 text-xl text-gray-800 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
                         </div>
+                        
                         <div class="w-full p-4 mb-4 rounded border border-solid border-gray-200 shadow bg-white">
                             <label for="{{ form.text.id_for_label }}" class="block mb-2 font-bold cursor-pointer">{{ form.text.label }}</label>
-                            {% render_field form.text class="appearance-none block w-full bg-gray-200 text-xl text-gray-800 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
+                            {% render_field form.text onkeyup="ShowLength(value);" class="appearance-none block w-full bg-gray-200 text-xl text-gray-800 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
+                            <p id="inputlength">/250</p>
```

Comment:
```suggestion
                            {% render_field form.text class="appearance-none block w-full bg-gray-200 text-xl text-gray-800 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
                            <p id="inputlength">/250</p>
```
Do not use inline JS handler, but rather create an event listener in your JS code:
```
document.getElementById("id_form-0-text").addEventListener("keyup", ShowLength);
```

Also, please keep in mind that there are multiple forms on the same page (one for each language), so just updating one form is no enough. We should update the values for each form separately.